tor/tor.go

```diff
@@ -125,7 +131,7 @@ func LookupSRV(service, proto, name, socksAddr, dnsServer string,
 	streamIsolation bool) (string, []*net.SRV, error) {
 
 	// Connect to the DNS server we'll be using to query SRV records.
-	conn, err := dial(dnsServer, socksAddr, streamIsolation)
+	conn, err := dial(dnsServer, socksAddr, streamIsolation, DefaultConnTimeout)
```

Comment:
`LookupSRV` should also accept the timeout parameter. It's invoked from the `DNSSeedBootstrapper`, which already knows of the configured value.

Reply:
fixed.